kolibri/content/test/test_content_app.py

```diff
@@ -182,14 +184,14 @@ def test_contentnode_granular_network_import(self):
         self.assertEqual(
             response.data, {
                 "pk": c1_id, "title": "root", "kind": "topic", "available": False,
-                "total_resources": 4, "on_device_resources": 0, "importable": True, "children": [
+                "total_resources": 2, "on_device_resources": 0, "importable": True, "children": [
                     {
                         "pk": c2_id, "title": "c1", "kind": "video", "available": False,
-                        "total_resources": 1, "on_device_resources": 0, "importable": True
+                        "total_resources": 2, "on_device_resources": 0, "importable": True
```

Comment:
I'm wondering why the `total_resources` of the node `c1` is 2 instead of 1. From my understanding, the node`c1` doesn't have any child contentnodes, so its `total_resources` is 1, which stands for itself as a video contentnode.